RequestService/RequestService.Repo/Repository.cs

```diff
@@ -803,23 +776,60 @@ private RequestSummary MapEFRequestToSummary(Request request)
                         Location = (Location)request.Shift.LocationId
                     };
                 }
+
+                List<JobBasic> jobBasics = new List<JobBasic>();
+                foreach (var job in request.Job)
+                {
+                    HelpMyStreet.Utils.Enums.SupportActivities sa = (HelpMyStreet.Utils.Enums.SupportActivities)job.SupportActivityId;
+                    if (sa.RequestType() == RequestType.Task)
+                    {
+                        jobBasics.Add(new JobSummary()
+                        {
+                            IsHealthCritical = job.IsHealthCritical,
+                            DueDate = job.DueDate,
+                            JobID = job.Id,
+                            VolunteerUserID = job.VolunteerUserId,
+                            JobStatus = (JobStatuses)job.JobStatusId,
+                            SupportActivity = (HelpMyStreet.Utils.Enums.SupportActivities)job.SupportActivityId,
+                            PostCode = job.NewRequest.PostCode,
+                            ReferringGroupID = job.NewRequest.ReferringGroupId,
+                            Groups = job.JobAvailableToGroup.Select(x => x.GroupId).ToList(),
+                            RecipientOrganisation = job.NewRequest.OrganisationName,
+                            DateStatusLastChanged = job.RequestJobStatus.Max(x => x.DateCreated),
+                            DueDays = Convert.ToInt32((job.DueDate.Date - DateTime.Now.Date).TotalDays),
+                            DateRequested = job.NewRequest.DateRequested,
+                            RequestorType = (RequestorType)job.NewRequest.RequestorType,
+                            Archive = job.NewRequest.Archive,
+                            DueDateType = (DueDateType)job.DueDateTypeId,
+                            RequestorDefinedByGroup = job.NewRequest.RequestorDefinedByGroup,
+                            RequestID = job.NewRequest.Id,
+                            RequestType = (RequestType)job.NewRequest.RequestType
+                        });
+                    }
+                    else
+                    {
+                        jobBasics.Add(new JobBasic()
```

Comment:
(Having done this, it would probably be better to use `switch` rather than `if / else`.   Default case could either add a JobBasic, or throw an exception - to future-proof for further RequestType values.)